.env

```
@@ -1,3 +1,3 @@
-DATABASE_URL=postgres://postgres:example@localhost/fileshare
+DATABASE_URL=postgresql://neondb_owner:npg_6lOnJpeNYHt0@ep-young-cell-agc57ajt.c-2.eu-central-1.aws.neon.tech/neondb?sslmode=require&channel_binding=require
 SALT=[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1]
 JWT_SECRET="eruifgaklhblfsgyuskghdsjkghfskghriousghjfaskguwoeuifghks"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)